docs/PARTITION_TABLE_EDITOR.md

```diff
@@ -0,0 +1,27 @@
+# Partition Table Editor UI for ESP-IDF
+
+Our VS Code Extension comes with UI for editing your [partition table](https://docs.espressif.com/projects/esp-idf/en/latest/esp32/api-guides/partition-tables.html) and flash it to your chip, instead of editing the csv files directly we present you with comfortable UI where you can edit an existing partition table or create a new one of your choosing.
+
+## Prerequisites
+
+- ESP-IDF `>=v4.x`
+- IDF VSCode extension version `>=0.6.0`
+- ESP Wrover Kit (optional for flashing the modified partition table)
+
+## Steps
+
+- Open any IDF where you want to have custom partition table.
+- Set `CONFIG_PARTITION_TABLE_CUSTOM` in the menuconfig and set your partition table csv file name.
+- Launch command palate using <kbd>F1</kbd>, and select `Open Partition Table Editor`.
+  > If there is no partition table file created based on the `` config, set via menuconfig, we will create an empty file.
```

Comment:
based on the `CONFIG_PARTITION_TABLE_CUSTOM` config set via menuconfig or SDK Configuration Editor